### PR TITLE
Script command follow-up

### PR DESCRIPTION
**Pull request by ravepossum**
This is a follow-up to #224 to finish up some feedback. So far it just documents the `MessageVar` macro and addresses a few points of feedback (`destVarID` as argument name for macros that return the result in a var), but I can add on any further feedback that wasn't addressed in the last PR.

### CODE REVIEW

File: asm/macros/scrcmd.inc [hunk 1]
```diff
@@ -14,7 +14,7 @@
     .short 2
     .endm
```
Comment (cbt6) [inline asm/macros/scrcmd.inc:16]:
```suggestion
    .short 2
    .endm

```

File: asm/macros/scrcmd.inc [hunk 2]
```diff
@@ -14,7 +14,7 @@
...
     
-    .macro WaitFrames frames, countdown
+    .macro WaitTime frames, countdown
```
Comment (cbt6) [inline asm/macros/scrcmd.inc:17]:
```suggestion
    .macro WaitTime frames, countdownVarID
```

**Review by cbt6 — COMMENTED** [review]
- `FieldSystem_GetVar` -> `FieldSystem_GetVarPointer`
- Use constants for args in `PlayFanfare`

**Comment by ravepossum** [comment]
Should be ready for another look whenever now.

File: src/scrcmd.c [hunk 3]
```diff
@@ -2291,50 +2291,43 @@ static BOOL ScrCmd_02F (ScriptContext * ctx)
...
-static BOOL sub_02040190 (ScriptContext * ctx)
+static BOOL ScriptContext_CheckABPress (ScriptContext * ctx)
 {
-    if (gCoreSys.pressedKeys & (PAD_BUTTON_A | PAD_BUTTON_B)) {
-        return 1;
-    }
-
-    return 0;
+    // this doesn't match using == TRUE or leaving off a comparison entirely
+    return (gCoreSys.pressedKeys & (PAD_BUTTON_A | PAD_BUTTON_B)) != FALSE;
```
Comment (lhearachel) [inline src/scrcmd.c:2303]: **general comment**: that makes sense to me, because this is a bitwise `AND`.

**Review by lhearachel — APPROVED** [review]

**Pull request merged**